{
	"documenttitle": {
		"text": "Red Hat Security Advisory: Red Hat Integration Camel for Spring Boot 3.18.3 Patch 2 release",
		"lang": "en"
	},
	"DocumentType": "Security Advisory",
	"documentpublisher": {
		"type": "Vendor",
		"ContactDetails": "secalert@redhat.com",
		"IssuingAuthority": "Red Hat Product Security"
	},
	"documenttracking": {
		"identification": {
			"ID": "RHSA-2023:3641"
		},
		"Status": "Final",
		"Version": "1",
		"revisionhistory": {
			"revision": {
				"Number": "1",
				"Date": "2023-06-15T15:23:00Z",
				"Description": "Current version"
			}
		},
		"InitialReleaseDate": "2023-06-15T15:23:00Z",
		"CurrentReleaseDate": "2023-06-15T15:23:00Z",
		"generator": {
			"Engine": "Red Hat SDEngine 3.17.1",
			"Date": "2023-06-15T20:06:00Z"
		}
	},
	"documentnotes": {
		"note": [
			{
				"text": "\nCamel for Spring Boot 3.18.3 Patch 2 release and security update is now available.\n\nRed Hat Product Security has rated this update as having an impact of Important. A Common Vulnerability Scoring System (CVSS) base score, which gives a detailed severity rating, is available for each vulnerability from the CVE link(s) in the References section.\n    ",
				"title": "Topic",
				"type": "Summary",
				"ordinal": "1",
				"lang": "en"
			},
			{
				"text": "\nThis release of Camel for Spring Boot 3.18.3.P2 serves as a replacement for Camel for Spring Boot 3.18.3.P1 and includes bug fixes and enhancements, which are documented in the Release Notes linked in the References. The purpose of this text-only errata is to inform you about the security issues fixed.\n\n* spring-boot: Spring Boot Welcome Page DoS Vulnerability (CVE-2023-20883)\n\n* woodstox-core: woodstox to serialise XML data was vulnerable to Denial of Service attacks (CVE-2022-40152)\n\n* xstream: Xstream to serialise XML data was vulnerable to Denial of Service attacks (CVE-2022-40156)\n\n* dev-java-snakeyaml: dev-java/snakeyaml: DoS via stack overflow (CVE-2022-41854)\n\n* snakeyaml: Denial of Service due to missing nested depth limitation for collections (CVE-2022-25857)\n\n* sshd-common: mina-sshd: Java unsafe deserialization vulnerability (CVE-2022-45047)\n\n* jettison: Uncontrolled Recursion in JSONArray (CVE-2023-1436)\n\n* json-smart: Uncontrolled Resource Consumption vulnerability in json-smart (Resource Exhaustion) (CVE-2023-1370)\n\n* jackson-databind: use of deeply nested arrays (CVE-2022-42004)\n\n* jackson-databind: deep wrapper array nesting wrt UNWRAP_SINGLE_VALUE_ARRAYS (CVE-2022-42003)\n\n* snakeyaml: Uncaught exception in org.yaml.snakeyaml.composer.Composer.composeSequenceNode (CVE-2022-38749)\n\n* snakeyaml: Uncaught exception in org.yaml.snakeyaml.constructor.BaseConstructor.constructObject (CVE-2022-38750)\n\n* snakeyaml: Uncaught exception in java.base/java.util.regex.Pattern.match (CVE-2022-38751)\n\n* snakeyaml: Uncaught exception in java.base/java.util.ArrayList.hashCode (CVE-2022-38752)\n\n* snakeyaml: Denial of Service due to missing nested depth limitation for collections (CVE-2022-25857)\n\n* CXF: Apache CXF: directory listing / code exfiltration (CVE-2022-46363)\n\n* CXF: Apache CXF: SSRF Vulnerability (CVE-2022-46364)\n\nFor more details about the security issue(s), including the impact, a CVSS score, acknowledgments, and other related information, refer to the CVE page(s) listed in the References section.\n    ",
				"title": "Details",
				"type": "General",
				"ordinal": "2",
				"lang": "en"
			},
			{
				"text": "This content is licensed under the Creative Commons Attribution 4.0 International License (https://creativecommons.org/licenses/by/4.0/). If you distribute this content, or a modified version of it, you must provide attribution to Red Hat Inc. and provide a link to the original.",
				"title": "Terms of Use",
				"type": "Legal Disclaimer",
				"ordinal": "3",
				"lang": "en"
			}
		]
	},
	"documentdistribution": {
		"text": "Copyright Â© 2023 Red Hat, Inc. All rights reserved.",
		"lang": "en"
	},
	"aggregateseverity": {
		"text": "Important",
		"namespace": "https://access.redhat.com/security/updates/classification/"
	},
	"documentreferences": {
		"reference": [
			{
				"type": "Self",
				"URL": "https://access.redhat.com/errata/RHSA-2023:3641",
				"Description": "https://access.redhat.com/errata/RHSA-2023:3641"
			},
			{
				"type": "External",
				"URL": "https://access.redhat.com/security/updates/classification/#important",
				"Description": "https://access.redhat.com/security/updates/classification/#important"
			},
			{
				"type": "External",
				"URL": "https://access.redhat.com/jbossnetwork/restricted/listSoftware.html?downloadType=distributions&product=red.hat.integration&version=2023-Q2",
				"Description": "https://access.redhat.com/jbossnetwork/restricted/listSoftware.html?downloadType=distributions&product=red.hat.integration&version=2023-Q2"
			}
		]
	},
	"vulnerability": [
		{
			"ordinal": "1",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "A flaw was found in the org.yaml.snakeyaml package. This flaw allows an attacker to cause a denial of service (DoS) due to missing nested depth limitation for collections.",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2022-08-30T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2022-25857",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nBefore applying this update, make sure all previously released errata\nrelevant to your system have been applied.\n\nFor details on how to apply this update, refer to:\n\nhttps://access.redhat.com/articles/11258\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2023:3641"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://www.cve.org/CVERecord?id=CVE-2022-25857",
						"Description": "https://www.cve.org/CVERecord?id=CVE-2022-25857"
					},
					{
						"URL": "https://nvd.nist.gov/vuln/detail/CVE-2022-25857",
						"Description": "https://nvd.nist.gov/vuln/detail/CVE-2022-25857"
					},
					{
						"URL": "https://bitbucket.org/snakeyaml/snakeyaml/issues/525",
						"Description": "https://bitbucket.org/snakeyaml/snakeyaml/issues/525"
					},
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2022-25857",
						"Description": "CVE-2022-25857"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=2126789",
						"Description": "bz#2126789: Denial of Service due to missing nested depth limitation for collections"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Important"
				}
			},
			"DiscoveryDate": "2022-09-14T00:00:00Z",
			"productstatuses": {
				"status": {
					"ProductID": null
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "2",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "A flaw was found in the snakeyaml package due to a stack-overflow in parsing YAML files. By persuading a victim to open a specially-crafted file, a remote attacker could cause the application to crash, resulting in a denial of service.",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2022-09-05T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2022-38749",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nBefore applying this update, make sure all previously released errata\nrelevant to your system have been applied.\n\nFor details on how to apply this update, refer to:\n\nhttps://access.redhat.com/articles/11258\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2023:3641"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://www.cve.org/CVERecord?id=CVE-2022-38749",
						"Description": "https://www.cve.org/CVERecord?id=CVE-2022-38749"
					},
					{
						"URL": "https://nvd.nist.gov/vuln/detail/CVE-2022-38749",
						"Description": "https://nvd.nist.gov/vuln/detail/CVE-2022-38749"
					},
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2022-38749",
						"Description": "CVE-2022-38749"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=2129706",
						"Description": "bz#2129706: Uncaught exception in org.yaml.snakeyaml.composer.Composer.composeSequenceNode"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2022-09-26T00:00:00Z",
			"productstatuses": {
				"status": {
					"ProductID": null
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "3",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "A flaw was found in the snakeyaml package due to a stack-overflow in parsing YAML files. By persuading a victim to open a specially-crafted file, a remote attacker could cause the application to crash.",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2022-09-05T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2022-38750",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nBefore applying this update, make sure all previously released errata\nrelevant to your system have been applied.\n\nFor details on how to apply this update, refer to:\n\nhttps://access.redhat.com/articles/11258\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2023:3641"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://www.cve.org/CVERecord?id=CVE-2022-38750",
						"Description": "https://www.cve.org/CVERecord?id=CVE-2022-38750"
					},
					{
						"URL": "https://nvd.nist.gov/vuln/detail/CVE-2022-38750",
						"Description": "https://nvd.nist.gov/vuln/detail/CVE-2022-38750"
					},
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2022-38750",
						"Description": "CVE-2022-38750"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=2129707",
						"Description": "bz#2129707: Uncaught exception in org.yaml.snakeyaml.constructor.BaseConstructor.constructObject"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2022-09-26T00:00:00Z",
			"productstatuses": {
				"status": {
					"ProductID": null
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "4",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "A flaw was found in the snakeyaml package due to a stack-overflow in parsing YAML files. By persuading a victim to open a specially-crafted file, a remote attacker could cause the application to crash.",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2022-09-05T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2022-38751",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nBefore applying this update, make sure all previously released errata\nrelevant to your system have been applied.\n\nFor details on how to apply this update, refer to:\n\nhttps://access.redhat.com/articles/11258\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2023:3641"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://www.cve.org/CVERecord?id=CVE-2022-38751",
						"Description": "https://www.cve.org/CVERecord?id=CVE-2022-38751"
					},
					{
						"URL": "https://nvd.nist.gov/vuln/detail/CVE-2022-38751",
						"Description": "https://nvd.nist.gov/vuln/detail/CVE-2022-38751"
					},
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2022-38751",
						"Description": "CVE-2022-38751"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=2129709",
						"Description": "bz#2129709: Uncaught exception in java.base/java.util.regex.Pattern$Ques.match"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2022-09-26T00:00:00Z",
			"productstatuses": {
				"status": {
					"ProductID": null
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "5",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "A flaw was found in the snakeyaml package due to a stack-overflow in parsing YAML files. By persuading a victim to open a specially-crafted file, a remote attacker could cause the application to crash.",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2022-09-05T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2022-38752",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nBefore applying this update, make sure all previously released errata\nrelevant to your system have been applied.\n\nFor details on how to apply this update, refer to:\n\nhttps://access.redhat.com/articles/11258\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2023:3641"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://www.cve.org/CVERecord?id=CVE-2022-38752",
						"Description": "https://www.cve.org/CVERecord?id=CVE-2022-38752"
					},
					{
						"URL": "https://nvd.nist.gov/vuln/detail/CVE-2022-38752",
						"Description": "https://nvd.nist.gov/vuln/detail/CVE-2022-38752"
					},
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2022-38752",
						"Description": "CVE-2022-38752"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=2129710",
						"Description": "bz#2129710: Uncaught exception in java.base/java.util.ArrayList.hashCode"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2022-09-26T00:00:00Z",
			"productstatuses": {
				"status": {
					"ProductID": null
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "6",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "A flaw was found in the  FasterXML/woodstox package. This flaw allows an attacker to cause a denial of service (DoS) in its target via XML serialization. An attacker may benefit from the parser sending a malicious input that may cause a crash. This vulnerability is only relevant for users making use of the DTD parsing functionality.",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2022-09-16T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2022-40152",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nBefore applying this update, make sure all previously released errata\nrelevant to your system have been applied.\n\nFor details on how to apply this update, refer to:\n\nhttps://access.redhat.com/articles/11258\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2023:3641"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://www.cve.org/CVERecord?id=CVE-2022-40152",
						"Description": "https://www.cve.org/CVERecord?id=CVE-2022-40152"
					},
					{
						"URL": "https://nvd.nist.gov/vuln/detail/CVE-2022-40152",
						"Description": "https://nvd.nist.gov/vuln/detail/CVE-2022-40152"
					},
					{
						"URL": "https://github.com/advisories/GHSA-3f7h-mf4q-vrm4",
						"Description": "https://github.com/advisories/GHSA-3f7h-mf4q-vrm4"
					},
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2022-40152",
						"Description": "CVE-2022-40152"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=2134291",
						"Description": "bz#2134291: woodstox to serialise XML data was vulnerable to Denial of Service attacks"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2022-10-13T00:00:00Z",
			"productstatuses": {
				"status": {
					"ProductID": null
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "7",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "A flaw was found in the XStream package. This flaw allows an attacker to cause a denial of service (DoS) in its target via XML serialization.",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2022-09-16T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2022-40156",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nBefore applying this update, make sure all previously released errata\nrelevant to your system have been applied.\n\nFor details on how to apply this update, refer to:\n\nhttps://access.redhat.com/articles/11258\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2023:3641"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://www.cve.org/CVERecord?id=CVE-2022-40156",
						"Description": "https://www.cve.org/CVERecord?id=CVE-2022-40156"
					},
					{
						"URL": "https://nvd.nist.gov/vuln/detail/CVE-2022-40156",
						"Description": "https://nvd.nist.gov/vuln/detail/CVE-2022-40156"
					},
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2022-40156",
						"Description": "CVE-2022-40156"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=2134288",
						"Description": "bz#2134288: Xstream to serialise XML data was vulnerable to Denial of Service attacks"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2022-10-13T00:00:00Z",
			"productstatuses": {
				"status": {
					"ProductID": null
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "8",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "Those using Snakeyaml to parse untrusted YAML files may be vulnerable to Denial of Service attacks (DOS). If the parser is running on user supplied input, an attacker may supply content that causes the parser to crash by stack overflow. This effect may support a denial of service attack.",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2022-11-13T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2022-41854",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nBefore applying this update, make sure all previously released errata\nrelevant to your system have been applied.\n\nFor details on how to apply this update, refer to:\n\nhttps://access.redhat.com/articles/11258\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2023:3641"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://www.cve.org/CVERecord?id=CVE-2022-41854",
						"Description": "https://www.cve.org/CVERecord?id=CVE-2022-41854"
					},
					{
						"URL": "https://nvd.nist.gov/vuln/detail/CVE-2022-41854",
						"Description": "https://nvd.nist.gov/vuln/detail/CVE-2022-41854"
					},
					{
						"URL": "https://bitbucket.org/snakeyaml/snakeyaml/issues/543/stackoverflow-oss-fuzz-50355",
						"Description": "https://bitbucket.org/snakeyaml/snakeyaml/issues/543/stackoverflow-oss-fuzz-50355"
					},
					{
						"URL": "https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=50355",
						"Description": "https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=50355"
					},
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2022-41854",
						"Description": "CVE-2022-41854"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=2151988",
						"Description": "bz#2151988: DoS via stack overflow"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2022-12-08T00:00:00Z",
			"productstatuses": {
				"status": {
					"ProductID": null
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "9",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "A flaw was found in FasterXML jackson-databind. This issue could allow an attacker to benefit from resource exhaustion when the UNWRAP_SINGLE_VALUE_ARRAYS feature is enabled due to unchecked primitive value deserializers to avoid deep wrapper array nesting.",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2022-10-02T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2022-42003",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nBefore applying this update, make sure all previously released errata\nrelevant to your system have been applied.\n\nFor details on how to apply this update, refer to:\n\nhttps://access.redhat.com/articles/11258\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2023:3641"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://www.cve.org/CVERecord?id=CVE-2022-42003",
						"Description": "https://www.cve.org/CVERecord?id=CVE-2022-42003"
					},
					{
						"URL": "https://nvd.nist.gov/vuln/detail/CVE-2022-42003",
						"Description": "https://nvd.nist.gov/vuln/detail/CVE-2022-42003"
					},
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2022-42003",
						"Description": "CVE-2022-42003"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=2135244",
						"Description": "bz#2135244: deep wrapper array nesting wrt UNWRAP_SINGLE_VALUE_ARRAYS"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2022-10-17T00:00:00Z",
			"productstatuses": {
				"status": {
					"ProductID": null
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "10",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "A flaw was found In FasterXML jackson-databind. This issue could allow an attacker to benefit from resource exhaustion due to the lack of a check in BeanDeserializer._deserializeFromArray to prevent the use of deeply nested arrays. An application is only vulnerable with certain customized choices for deserialization.",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2022-10-02T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2022-42004",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nBefore applying this update, make sure all previously released errata\nrelevant to your system have been applied.\n\nFor details on how to apply this update, refer to:\n\nhttps://access.redhat.com/articles/11258\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2023:3641"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://www.cve.org/CVERecord?id=CVE-2022-42004",
						"Description": "https://www.cve.org/CVERecord?id=CVE-2022-42004"
					},
					{
						"URL": "https://nvd.nist.gov/vuln/detail/CVE-2022-42004",
						"Description": "https://nvd.nist.gov/vuln/detail/CVE-2022-42004"
					},
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2022-42004",
						"Description": "CVE-2022-42004"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=2135247",
						"Description": "bz#2135247: use of deeply nested arrays"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2022-10-17T00:00:00Z",
			"productstatuses": {
				"status": {
					"ProductID": null
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "11",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "A flaw was found in Apache MINA SSHD, when using Java deserialization to load a serialized java.security.PrivateKey. An attacker could benefit from unsafe deserialization by inserting unsecured data that may affect the application or server.",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2022-11-16T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2022-45047",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nBefore applying this update, make sure all previously released errata\nrelevant to your system have been applied.\n\nFor details on how to apply this update, refer to:\n\nhttps://access.redhat.com/articles/11258\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2023:3641"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://www.cve.org/CVERecord?id=CVE-2022-45047",
						"Description": "https://www.cve.org/CVERecord?id=CVE-2022-45047"
					},
					{
						"URL": "https://nvd.nist.gov/vuln/detail/CVE-2022-45047",
						"Description": "https://nvd.nist.gov/vuln/detail/CVE-2022-45047"
					},
					{
						"URL": "https://www.mail-archive.com/dev@mina.apache.org/msg39312.html",
						"Description": "https://www.mail-archive.com/dev@mina.apache.org/msg39312.html"
					},
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2022-45047",
						"Description": "CVE-2022-45047"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=2145194",
						"Description": "bz#2145194: Java unsafe deserialization vulnerability"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Important"
				}
			},
			"DiscoveryDate": "2022-11-23T00:00:00Z",
			"productstatuses": {
				"status": {
					"ProductID": null
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "12",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "A vulnerability was found in Apache CXF that could allow an attacker to perform a remote directory listing or code exfiltration. This issue only applies when the CXFServlet is configured with both the static-resources-list and redirect-query-check attributes. These attributes are not supposed to be used together, so the issue can only occur if the CXF service is misconfigured.",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2022-12-13T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2022-46363",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nBefore applying this update, make sure all previously released errata\nrelevant to your system have been applied.\n\nFor details on how to apply this update, refer to:\n\nhttps://access.redhat.com/articles/11258\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2023:3641"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://www.cve.org/CVERecord?id=CVE-2022-46363",
						"Description": "https://www.cve.org/CVERecord?id=CVE-2022-46363"
					},
					{
						"URL": "https://nvd.nist.gov/vuln/detail/CVE-2022-46363",
						"Description": "https://nvd.nist.gov/vuln/detail/CVE-2022-46363"
					},
					{
						"URL": "https://lists.apache.org/thread/pdzo1qgyplf4y523tnnzrcm7hoco3l8c",
						"Description": "https://lists.apache.org/thread/pdzo1qgyplf4y523tnnzrcm7hoco3l8c"
					},
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2022-46363",
						"Description": "CVE-2022-46363"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=2155681",
						"Description": "bz#2155681: directory listing / code exfiltration"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2022-12-21T00:00:00Z",
			"productstatuses": {
				"status": {
					"ProductID": null
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "13",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "A SSRF vulnerability was found in Apache CXF. This issue occurs when parsing the href attribute of XOP:Include in MTOM requests, allowing an attacker to perform SSRF style attacks on webservices that take at least one parameter of any type.",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2022-12-13T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2022-46364",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nBefore applying this update, make sure all previously released errata\nrelevant to your system have been applied.\n\nFor details on how to apply this update, refer to:\n\nhttps://access.redhat.com/articles/11258\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2023:3641"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://www.cve.org/CVERecord?id=CVE-2022-46364",
						"Description": "https://www.cve.org/CVERecord?id=CVE-2022-46364"
					},
					{
						"URL": "https://nvd.nist.gov/vuln/detail/CVE-2022-46364",
						"Description": "https://nvd.nist.gov/vuln/detail/CVE-2022-46364"
					},
					{
						"URL": "https://cxf.apache.org/security-advisories.data/CVE-2022-46364.txt?version=1&modificationDate=1670944472739&api=v2",
						"Description": "https://cxf.apache.org/security-advisories.data/CVE-2022-46364.txt?version=1&modificationDate=1670944472739&api=v2"
					},
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2022-46364",
						"Description": "CVE-2022-46364"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=2155682",
						"Description": "bz#2155682: SSRF Vulnerability"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Important"
				}
			},
			"DiscoveryDate": "2022-12-21T00:00:00Z",
			"productstatuses": {
				"status": {
					"ProductID": null
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "14",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "A flaw was found in the json-smart package. This security flaw occurs when reaching a â[â or â{â character in the JSON input, and the code parses an array or an object, respectively. The 3PP does not have any limit to the nesting of such arrays or objects. Since nested arrays and objects are parsed recursively, nesting too many of them can cause stack exhaustion (stack overflow) and crash the software.",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2023-03-22T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2023-1370",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nBefore applying this update, make sure all previously released errata\nrelevant to your system have been applied.\n\nFor details on how to apply this update, refer to:\n\nhttps://access.redhat.com/articles/11258\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2023:3641"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://www.cve.org/CVERecord?id=CVE-2023-1370",
						"Description": "https://www.cve.org/CVERecord?id=CVE-2023-1370"
					},
					{
						"URL": "https://nvd.nist.gov/vuln/detail/CVE-2023-1370",
						"Description": "https://nvd.nist.gov/vuln/detail/CVE-2023-1370"
					},
					{
						"URL": "https://github.com/advisories/GHSA-493p-pfq6-5258",
						"Description": "https://github.com/advisories/GHSA-493p-pfq6-5258"
					},
					{
						"URL": "https://research.jfrog.com/vulnerabilities/stack-exhaustion-in-json-smart-leads-to-denial-of-service-when-parsing-malformed-json-xray-427633/",
						"Description": "https://research.jfrog.com/vulnerabilities/stack-exhaustion-in-json-smart-leads-to-denial-of-service-when-parsing-malformed-json-xray-427633/"
					},
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2023-1370",
						"Description": "CVE-2023-1370"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=2188542",
						"Description": "bz#2188542: Uncontrolled Resource Consumption vulnerability in json-smart (Resource Exhaustion)"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Important"
				}
			},
			"DiscoveryDate": "2023-04-21T00:00:00Z",
			"productstatuses": {
				"status": {
					"ProductID": null
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "15",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "A flaw was found in Jettison. Infinite recursion is triggered in Jettison when constructing a JSONArray from a Collection that contains a self-reference in one of its elements. This issue leads to a StackOverflowError exception being thrown.",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2023-03-22T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2023-1436",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nBefore applying this update, make sure all previously released errata\nrelevant to your system have been applied.\n\nFor details on how to apply this update, refer to:\n\nhttps://access.redhat.com/articles/11258\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2023:3641"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://www.cve.org/CVERecord?id=CVE-2023-1436",
						"Description": "https://www.cve.org/CVERecord?id=CVE-2023-1436"
					},
					{
						"URL": "https://nvd.nist.gov/vuln/detail/CVE-2023-1436",
						"Description": "https://nvd.nist.gov/vuln/detail/CVE-2023-1436"
					},
					{
						"URL": "https://research.jfrog.com/vulnerabilities/jettison-json-array-dos-xray-427911/",
						"Description": "https://research.jfrog.com/vulnerabilities/jettison-json-array-dos-xray-427911/"
					},
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2023-1436",
						"Description": "CVE-2023-1436"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=2182788",
						"Description": "bz#2182788: Uncontrolled Recursion in JSONArray"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2023-03-29T00:00:00Z",
			"productstatuses": {
				"status": {
					"ProductID": null
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "16",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "A flaw was found in Spring Boot, occurring prominently in Spring MVC with a reverse proxy cache. This issue requires Spring MVC to have auto-configuration enabled and the application to use Spring Boot's welcome page support, either static or templated, resulting in the application being deployed behind a proxy that caches 404 responses. This issue may cause a denial of service (DoS) attack.",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2023-05-18T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2023-20883",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nBefore applying this update, make sure all previously released errata\nrelevant to your system have been applied.\n\nFor details on how to apply this update, refer to:\n\nhttps://access.redhat.com/articles/11258\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2023:3641"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://www.cve.org/CVERecord?id=CVE-2023-20883",
						"Description": "https://www.cve.org/CVERecord?id=CVE-2023-20883"
					},
					{
						"URL": "https://nvd.nist.gov/vuln/detail/CVE-2023-20883",
						"Description": "https://nvd.nist.gov/vuln/detail/CVE-2023-20883"
					},
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2023-20883",
						"Description": "CVE-2023-20883"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=2209342",
						"Description": "bz#2209342: Spring Boot Welcome Page DoS Vulnerability"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Important"
				}
			},
			"DiscoveryDate": "2023-05-23T00:00:00Z",
			"productstatuses": {
				"status": {
					"ProductID": null
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		}
	],
	"producttree": {}
}
